3rdparty/python/user_reqs.lock

```diff
@@ -0,0 +1,42 @@
+// This lockfile was autogenerated by Pants. To regenerate, run:
+//
+//    ./pants generate-lockfiles --resolve=python-default
+//
+// --- BEGIN PANTS LOCKFILE METADATA: DO NOT EDIT OR REMOVE ---
+// {
+//   "version": 3,
+//   "valid_for_interpreter_constraints": [
+//     "CPython==3.9.15"
+//   ],
+//   "generated_with_requirements": [],
+//   "manylinux": "manylinux2014",
+//   "requirement_constraints": [],
+//   "only_binary": [],
+//   "no_binary": []
+// }
+// --- END PANTS LOCKFILE METADATA ---
+
+{
+  "allow_builds": true,
+  "allow_prereleases": false,
+  "allow_wheels": true,
+  "build_isolation": true,
+  "constraints": [],
+  "locked_resolves": [],
+  "path_mappings": {},
+  "pex_version": "2.1.108",
+  "pip_version": "20.3.4-patched",
```

Comment:
pip is at 22.3.1, why such an old version?

Reply:
This is the lock file autogenerated by pants